aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -365,10 +365,6 @@ module aptos_token::token {
         direct_transfer(sender, receiver, token_id, amount);
     }
 
-    public entry fun initialize_token_script(account: &signer) {
```

Comment:
`public entry fun initialize_token_script(_account: &signer) {
        abort 0;
    }`

Reply:
+1

Reply:
Okay, should be good to go!